### DISCUSSION TITLE
is anycable-mqtt avaliable?

### DISCUSSION THREAD

**Discussion by Davidzhu001** [anycable/anycable#182] [Q&A]
I asked Chat-GPT and he give me the suggestion of use anycable-mqtt for mqtt protocal!

I couldn't find it anywhere!
<img width="898" alt="image" src="https://user-images.githubusercontent.com/9436230/211752330-a9f8a808-a194-4473-b9db-c274abce9516.png">

**Comment by palkan** [comment]
Wow!

Looks like it's something from the future; haven't heard about `anycable-mqtt` 🙂
